src/Php54/bootstrap.php

```diff
@@ -16,7 +16,14 @@
         function trait_exists($class, $autoload = true) { return $autoload && class_exists($class, $autoload) && false; }
     }
     if (!function_exists('class_uses')) {
-        function class_uses($class, $autoload = true) { return $autoload && class_exists($class, $autoload) && false; }
+        function class_uses($class, $autoload = true)
+        {
+            if (!is_object($class) && $autoload && !class_exists($class, true)) {
```

Comment:
when autoload === false, we should still return false when the class does not exist. But when $class is an interface, we shoudld return array()... Could be done with:
```php
if (is_object($class) || class_exists($class, $autoload) || interface_exists($class, false)) {
    return array();
}

return false;
```